packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -208,10 +211,20 @@ class FlutterErrorDetails extends Diagnosticable {
     this.library = 'Flutter framework',
     this.context,
     this.stackFilter,
+    this.debugCreator,
```

Comment:
creator really seems like the wrong word here. It's just a way of collecting more information, right? Why isn't `informationCollector` able to handle this?

Reply:
Information collector will be placed at the end of error message. I think we would like to surface this information at early place before stack trace. there is another reason that the way we hook the describer and debug creator, i don't think we can do it in information collector.